.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature Request
+about: Suggest a prompt
+title: "[INSERT PROMPT HERE]"
+labels: enhancement
+assignees: stemaMSFT
+
+---
```

Comment:
What's the purpose of this? Is this metadata for github's issue template, or will this be actually exposed to the user?

Reply:
metadata for github's issue template. Users would not get exposed to it.